exchange/exchangetest/firstpartydata-amp-imp-ext-one-prebid-bidder.json

```diff
@@ -0,0 +1,133 @@
+{
+    "requestType": "amp",
```

Comment:
What's the significance of `requestType`? I did a quick search and it looks like this field is almost never used, and for the few places where it is used, it is always web.
It's also a little odd that we are considering this an AMP test since it is only testing the exchange since most of the AMP-specific logic occurs in the AMP endpoint handler before the exchange is called. Perhaps there isn't an easy way to test this so you're just verifying imp.ext.data is passed through?

Reply:
Discussed offline, but want to reiterate.
`requestType` is a parameter to json test framework that controls a request type (auction, amp, video) to test `HoldAuction` function. 
It is always web, because it was added for FPD and FPD was only supported for web. Now as we added FPD support for amp this test has `"requestType": "amp"`

This test has imp.ext.data in `incomingRequest` and we also expect in in `outgoingRequests`

I investigated possibility to add full amp test. It doesn't not check imp.ext.data.